### DISCUSSION TITLE
Tasmota.bin vs. Tasmota-ir.bin

### DISCUSSION THREAD

**Discussion by thomazffracarolli** [arendst/Tasmota#13879] [Q&A] [answered]
Dears, hope you are doing great!

Recently I bought a new fan and I was struggling to send IR command to power ON, power OFF, speed and timer function because when I tried to read the code using IR_RECV I was getting UNKNOWN protocol which could not be sent. 

_{“Protocol”:“Unknown”,“Bits”:32,“Data”:“0x0143226DB”}_

After a lot of tentatives I decided to change my firmware from standard 9.5.0 to the 10.0.0 IR version. In this version I was able to read the code (below):

_{"Protocol":"SYMPHONY","Bits":12,"Data":"0xD84","DataLSB":"0xB021","Repeat":0}_

**Now the challenge:** 

I notice that the firmware Tasmota-IR.bin is supporting SYMPHONY protocol while Tasmota.bin isn't. The issue is: I'm currently using a DIY board with have analog sensor (LDR), DHT11, contact sensor and IR RECV - means that I cannot use Tasmota-IR.bin due the sensors and other devices. 

Could you please support me? What should I do to have both? All function that we have in Tasmota.bin and all the IR library we have in Tasmota-IR.bin....

Thank you in advance,
Thomaz

**Comment by barbudor** [comment, marked as answer]
See how tasmota-ir.bin is built both from the '.ini' files and the tasmota_configuration files and bring that into your own user_override

https://tasmota.github.io/docs/Compile-your-build/

**Reply by thomazffracarolli** [reply]
Thank you for your comment! 
Works perfect.
